platform-hub-api/app/controllers/concerns/agents_initializer.rb

```diff
@@ -0,0 +1,12 @@
+module AgentsInitializer
+  extend ActiveSupport::Concern
+
+  def git_hub_agent_service
+    @git_hub_agent_service ||= Agents::GitHubAgentService.new(
```

Comment:
> Should we call those `agents` or `services`?

It does _seem_ redundant to use both terms, but 'agents' are intended to be a subset of services, that access external resources/services and often have super powers (like a token with full admin access, for example). I've tried to explain this in [the API server README](https://github.com/UKHomeOffice/platform-hub/tree/master/platform-hub-api#general-architectural-points-of-interest) – let me know if you think this needs improving.

> Should this be moved to global initializer?

I prefer not to have them be available globally, and instead, controllers (or any other services) that need access to these agents should `include` this concern instead. This way the developer had to explicitly choose access to these agents.